e2e/transform/transformer-config/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "dependencies": {
+    "babel-plugin-transform-flow-strip-types": "6.8.0"
```

Comment:
we're using babel 7 now, can you update this?

Reply:
good catch! Should use the preset as well, not the transform directly (to be consistent)